design/fast_finality.md

```diff
@@ -0,0 +1,273 @@
+# Fast finality design
+
+## Scope
+
+The introduction of a single sequencer to improve cost-per-transaction during the bootstrapping phase, as 
+described in the [Bootstrapping Strategy design doc](./Bootstrapping_strategy.md).
+
+## Glossary
+
+* *Light batch*: A set of transactions produced by the designated sequencer, considered to be soft-finalised in that 
+  order
+
+## Requirements
+
+* Finality
+  * Transaction *soft* finality (finality guaranteed by the sequencer) is achieved in under one second
+  * There is eventual transaction *hard* finality (finality guaranteed by the L1)
+  * The sequencer is incentivised to achieve hard finality on an agreed cadence
+  * The sequencer is incentivised to hard-finalise transactions in the same order they are soft-finalised
```

Comment:
"strongly incentivised".
The next point is equivalent. I would merge them.

Reply:
Also, there is no mention of the stake and the slashing.

Reply:
I updated the requirement and merged the two.

The slashing mechanism is listed under future work. I was hoping to leave it out of this design, but I can try and add the design for that?

Reply:
Added a staking and slashing section.